### PR TITLE
[stable/airflow] release 7.11.1

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR:
* cleans up the documentation introduced in #23802
* cleans up some YAML

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@gsemet can you please review?

**Comment by gsemet** [comment]
look great thanks

/lgtm

File: stable/airflow/templates/scheduler/scheduler-deployment.yaml [hunk 1]
```diff
@@ -309,9 +309,12 @@ spec:
         {{- if or (.Values.scheduler.connections) (.Values.scheduler.existingSecretConnections) }}
         - name: connections
           secret:
-            secretName: {{ if .Values.scheduler.existingSecretConnections }}{{ .Values.scheduler.existingSecretConnections }}
-                        {{- else }}{{ include "airflow.fullname" . }}-connections
-                        {{- end }}
+            secretName: |
```
Comment (yoavo-datricks) [inline stable/airflow/templates/scheduler/scheduler-deployment.yaml:312]: Hi. I think this part is causing me an issue when trying to load an existing secret.
helm appends a new line at the end and now the pod is looking for the secret "airflow-connections\n"
Please have a look

Comment (thesuperzapper) [reply stable/airflow/templates/scheduler/scheduler-deployment.yaml:312]: @yoavo-datricks I have raised #23858 to fix this

Comment (thesuperzapper) [reply stable/airflow/templates/scheduler/scheduler-deployment.yaml:312]: @yoavo-datricks this should be fixed in `7.11.3`

Comment (yoavo-datricks) [reply stable/airflow/templates/scheduler/scheduler-deployment.yaml:312]: @thesuperzapper It's working, thanks!
BTW, using this script to create an initial admin user as well, super useful!

**Pull request merged**